HW03/Dzmitry_Neverov/mentor.rb

```diff
@@ -0,0 +1,22 @@
+class Mentor
+  attr_reader :name, :surname, :notification
+  attr_accessor :all_hw
```

Comment:
Please, use full and clear name everywhere 
```suggestion
  attr_accessor :all_homeworks
```